src/collision/cm_material.h

```diff
@@ -73,8 +89,7 @@ typedef enum {
 
 typedef struct cm_stage_s {
 	uint32_t flags;
-	char image[MAX_QPATH];
-	int32_t image_index;
+	cm_asset_t asset;
```

Comment:
I like the idea of a common asset class. Might make cgame easier to manage down the road.